proto/provenance/name/v1/tx.proto

```diff
@@ -46,3 +51,18 @@ message MsgDeleteNameRequest {
 
 // MsgDeleteNameResponse defines the Msg/DeleteName response type.
 message MsgDeleteNameResponse {}
+
+// MsgCreateRootNameProposal defines an sdk.Msg type to create a new root name
+// that is controlled by a given owner and optionally restricted to the owner
+// for the sole creation of sub names.
+message MsgCreateRootNameRequest {
+  option (cosmos.msg.v1.signer)    = "authority";
+
+  string authority   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string name        = 2;
```

Comment:
I know we have a record object that includes these 3 fields. Maybe you want to use that instead.